src/components/CreateSkill/CreateSkill.js

```diff
@@ -352,6 +343,119 @@ export default class CreateSkill extends React.Component {
             width: '100%',
             padding: '10px'
         };
+        if(!cookies.get('loggedIn')) {
+            const titleStyle = {
```

Comment:
The logic isn't quite working. You need to define an else statement for which the visibility should be hidden. I was able to see the save button without login. Please check with others too.
Thanks for the PR.

Reply:
Can you check again? Its' working fine for me, ie. the save button is not visible without login